protocol/communities/manager.go

```diff
@@ -680,10 +680,27 @@ func (m *Manager) Joined() ([]*Community, error) {
 	return m.persistence.JoinedCommunities(&m.identity.PublicKey)
 }
 
+func (m *Manager) UpdateLastOpenedAt(communityID types.HexBytes) error {
+	return m.persistence.UpdateLastOpenedAt(communityID)
+}
+
 func (m *Manager) Spectated() ([]*Community, error) {
 	return m.persistence.SpectatedCommunities(&m.identity.PublicKey)
 }
 
+func (m *Manager) CommunityUpdateLastOpenedAt(communityID string) (*Community, error) {
+	community, err := m.GetByIDString(communityID)
+	if err != nil {
+		return nil, err
+	}
+	err = m.UpdateLastOpenedAt(community.ID())
```

Comment:
I believe we don't need extra public function `UpdateLastOpenedAt`, it is used only here.

```suggestion
	err = m.persistence.UpdateLastOpenedAt(communityID)
```

Reply:
Done